modules/standard/FileSystem.chpl

```diff
@@ -59,7 +59,7 @@ proc locale.chdir(name: string) {
               question.  See description of the provided constants for
               potential values.
 */
-proc chmod(out err: syserr, name: string, mode: int) {
+proc chmod(out error: syserr, name: string, mode: int) {
   extern proc chpl_fs_chmod(name: c_string, mode: int): syserr;
 
   err = chpl_fs_chmod(name.c_str(), mode);
```

Comment:
Let me know if testing didn't detect that (for file functions added before a couple of weeks ago).  I'm going to add more tests of all the new functions, and it'd be helpful for me to know if I need to add tests for one of the older ones.